pom.xml

```diff
@@ -145,6 +145,12 @@
                 <artifactId>jaxb-api</artifactId>
                 <version>2.3.1</version>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk -->
+            <dependency>
+                <groupId>com.amazonaws</groupId>
+                <artifactId>aws-java-sdk-simpleworkflow</artifactId>
+                <version>1.11.86</version>
```

Comment:
if the aws-java-sdk-core and this is likely to remain the same version, use a property

Reply:
Added a property: https://github.com/dockstore/dockstore-cli/pull/99/commits/c1864bef14187d41e149c5e582cff59b82c007de